### PR TITLE
Contactform session token

### PR DESCRIPTION
**Pull request by mmcgowanBOS**

### CODE REVIEW

File: docroot/modules/custom/bos_components/modules/bos_email/src/Controller/TokenOps.php [hunk 1]
```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Drupal\bos_email\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Core\Site\Settings;
+use Drupal\bos_email\Templates\Contactform;
```
Comment (davidrkupton) [inline docroot/modules/custom/bos_components/modules/bos_email/src/Controller/TokenOps.php:7]: Dont htink you use this? (line 7)

Comment (davidrkupton) [reply docroot/modules/custom/bos_components/modules/bos_email/src/Controller/TokenOps.php:7]: or line 6?

File: docroot/modules/custom/bos_components/modules/bos_email/src/Controller/PostmarkAPI.php [hunk 2]
```diff
@@ -45,6 +46,61 @@ public static function create(ContainerInterface $container) {
     );
   }
 
+
+  /**
+   * Check / set valid session token.
+   *
+   */
+  public function token(string $operation) {
+    /*$session = \Drupal::request()->getSession();
```
Comment (davidrkupton) [inline docroot/modules/custom/bos_components/modules/bos_email/src/Controller/PostmarkAPI.php:55]: maybe remove those comments

**Review by davidrkupton — APPROVED** [review]
LGTM!

File: docroot/modules/custom/bos_components/modules/bos_email/src/Controller/PostmarkAPI.php (hunk 2 above)
Comment (mmcgowanBOS) [reply docroot/modules/custom/bos_components/modules/bos_email/src/Controller/PostmarkAPI.php:55]: Done.

File: docroot/modules/custom/bos_components/modules/bos_email/src/Controller/TokenOps.php (hunk 1 above)
Comment (mmcgowanBOS) [reply docroot/modules/custom/bos_components/modules/bos_email/src/Controller/TokenOps.php]: Yup, removed both.

**Pull request merged**